src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -805,14 +811,20 @@ export function buildTooltipForTwoAttributesFactory(
     viewByParentAttribute: IUnwrappedAttributeHeadersWithItems,
     config: IChartConfig = {},
 ): ITooltipFactory {
-    const { separators } = config;
+    const { separators, stackMeasuresToPercent = false } = config;
+
     const formatValue = (val: number, format: string) => {
         return colors2Object(numberFormat(val, format, undefined, separators));
     };
 
-    return (point: IPoint) => {
+    return (point: IPoint, percentageValue?: number): string => {
         const category: ICategory = point.category;
-        const formattedValue = customEscape(formatValue(point.y, point.format).label);
+
+        const isNotStackToPercent = stackMeasuresToPercent === false || isNil(percentageValue);
+        const formattedValue = isNotStackToPercent
+            ? customEscape(formatValue(point.y, point.format).label)
+            : percentFormatter(percentageValue);
```

Comment:
Seems this block is similar to line 784-787. is it possible to separate into new function to reduce duplicate code?

Reply:
fixed